chia/wallet/wallet_state_manager.py

```diff
@@ -1540,15 +1537,15 @@ async def coin_added(
                     wallet_id=wallet_id,
                     sent_to=[],
                     trade_id=None,
-                    type=uint32(TransactionType.INCOMING_TX.value),
+                    type=uint32(tx_type),
```

Comment:
It still looks like `tx_type` changed *type* in this patch. it used to be `int` and now it's `TransactionType`. Casting this enum to `uint32`, is that really the same thing as asking for `.value`? I would expect that the enum itself might give you the *rank*, not the value assigned to the enum.

Reply:
Why don't you just try it in the console if you are concerned about it? You will see its working. I guess thats the magic of `IntEnum` what `TransactionType` is like i said here https://github.com/Chia-Network/chia-blockchain/pull/14852#discussion_r1143699640 :) 

Also, while `mypy` accepts both, PyCharm even complains about using `.value` here.